components/script/dom/windowproxy.rs

```diff
@@ -590,6 +608,96 @@ impl WindowProxy {
     }
 }
 
+// https://html.spec.whatwg.org/multipage/#concept-window-open-features-tokenize
+fn tokenize_open_features(features: DOMString) -> HashMap<String, String> {
+    let is_feature_sep = |c: char| ['\t', '\n', '\x0c', '\r', ' ', '=', ','].contains(&c);
+    // Step 1
+    let mut tokenized_features = HashMap::new();
```

Comment:
While the step 1 in spec said `Let tokenizedFeatures be a new ordered map.`, maybe we need to use `IndexMap` instead 🤔?